css/index.css

```diff
@@ -101,7 +101,7 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
```

Comment:
For this area, you can make use of parent/child relationships.  You can remove all the .box1 .box2 etc. from this.  See me comments in the index.html file.